src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -88,6 +88,9 @@ internal static class FeatureOnOffOptions
             nameof(FeatureOnOffOptions), nameof(AutomaticallyCompleteStatementOnSemicolon), defaultValue: true,
             storageLocations: new RoamingProfileStorageLocation($"TextEditor.{nameof(AutomaticallyCompleteStatementOnSemicolon)}"));
 
+        public static readonly Option2<bool> SkipAnalyzersForImplicitlyTriggeredBuilds = new(
```

Comment:
This needs to be added in [AutomationObject](https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/CSharp/Impl/Options/AutomationObject/AutomationObject.OnOff.cs) to get persisted correctly when the user imports/exports VS settings.

Reply:
Thanks @Youssef1313!